common/lib/xmodule/xmodule/tests/test_content.py

```diff
@@ -173,3 +176,239 @@ def test_static_content_write_js(self):
         js_file_paths = [file_path for file_path in js_file_paths if os.path.basename(file_path).startswith('000-')]
         self.assertEqual(len(js_file_paths), 1)
         self.assertIn("XModule.Descriptor = (function () {", open(js_file_paths[0]).read())
+
+
+@ddt.ddt
+class CanonicalContentTest(ModuleStoreTestCase):
```

Comment:
FWIW, you might want to try changing this to a (`SharedModuleStoreTestCase`)[http://swampcastle.org/2015/07/28/writing-faster-modulestore-tests.html] now that you've got so many `ddt.data` entries. Might save a decent chunk of time on test runs.